Gruntfile.js

```diff
@@ -144,7 +144,7 @@ module.exports = function(grunt) {
         ],
         options: {
           helpers: ['dist/*.js', 'dist/*.json'],
-          keepRunner: false
+          keepRunner: true
```

Comment:
Regarding why 10 seconds to detect a server ping, that was a somewhat
arbitrary choice done just to be safe.  In practice from my computer the
pings return almost instantly.  I could change this to 1 second, but it
would be pretty bad if we got some false disconnect alarms

On Fri Feb 06 2015 at 7:05:34 PM Raymond Cheng <notifications@github.com>
wrote:

> In Gruntfile.js
> <https://github.com/freedomjs/freedom-social-xmpp/pull/93#discussion_r24282076>
> :
>
> > @@ -144,7 +144,7 @@ module.exports = function(grunt) {
> >          ],
> >          options: {
> >            helpers: ['dist/*.js', 'dist/*.json'],
> > -          keepRunner: false
> > +          keepRunner: true
>
> revert?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom-social-xmpp/pull/93/files#r24282076>
> .
>